src/Runtime/OMTensorHelper.h

```diff
@@ -108,7 +108,7 @@ OMTensor *omTensorCreateWithRandomData(
  * @return typed element by reference at the offset computed by the index array.
  */
 template <typename T>
-T &omTensorGetElem(OMTensor *omt, std::vector<int64_t> indexes);
+T &omTensorGetElem(const OMTensor *omt, std::vector<int64_t> indexes);
```

Comment:
Hi @AlexandreEichenberger. The proposed declaration prevents `*omt` to be assigned to. That is, prevents the function to modify the memory location pointed to by `omt`. That memory location can only be read. 

Here is a simple example that clarifies the semantics:
```
int foo(const int *pi) {
  *pi = 3; // wrong: cannot assing the memory pointed to by pi.
  pi++; //ok: can change where pi points to (need "const int * const pi") to prevent this.
  return *pi;
}
```